### PR TITLE
Adding components to the mno-ui-elements

### PR DESCRIPTION
**Pull request by clemoberti**
I added a couple of components to the mno-ui-elements library.
Note that mno-admin and mno-kpi are the only components that have been tested and documented at the moment.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.project
+.settings
+
+# Ignore node stuff
+node_modules/
+npm-debug.log
+libpeerconnection.log
+
+# OS-specific
+.DS_Store
```
Comment (alexnoox) [inline .gitignore]: Duplicate

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+## Mno-ui-elements requirements
+* Mno-ui-elements requires **Bootstrap CSS** version 3.x or higher and it has been tested with Bootstrap CSS 3.3.6.
+* Mno-ui-elements requires **Font Awesome** version 4.x or higher and it has been tested with font-awesome 4.4.0.
+
+### Downloading the library
+
+* Add the following depency to your bower.json or package.json
+
+```json
+"mno-enterprise-angular": "git@github.com:maestrano/mno-ui-elements.git#master"
```
Comment (alexnoox) [inline README.md:15]: Note: We will have to register this lib on bower

File: src/components/admin/readme.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+You may want to use the default layout provided by mno-admin-nav. 
+
+* `.sidebar-title` -
+  Add this class to one of your list element to give it a title style.
+  
+* `.sidebar-list` -
+  Add this class to your list element to apply a default style.
+  
+  * `.menu-icon` - Add this class to a font awesome icon to help identifying the content.
```
Comment (alexnoox) [inline src/components/admin/readme.md:48]: Align with other list elements

File: src/components/kpi/kpi.component.coffee [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  transclude: true,
+  controllerAs: "vm",
+  bindings: {
+    description: '@',
+    loading: '=',
+    value: '@',
+    icon: '@?',
+    unit: '@?',
+    bgColor: '@?',
+    color:'@?',
```
Comment (alexnoox) [inline src/components/kpi/kpi.component.coffee]: The colors should be defined using less variables.

File: src/components/kpi/kpi.less [hunk 5]
```diff
@@ -0,0 +1,79 @@
+mno-kpi {
+
```
Comment (alexnoox) [inline src/components/kpi/kpi.less]: No need of an empty line here

File: src/components/loadingEllipsis/readme.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+The **loading ellipsis**  provides a simple animation with three circles to inform the user about another component loading.
+
+''' html
+<mno-loading-ellipsis ng-show="isVisible">
+</mno-loading-ellipsis>
```
Comment (alexnoox) [inline src/components/loadingEllipsis/readme.md]: If there is no content, write the elements on the same line:

`<mno-loading-ellipsis ng-show="isVisible"></mno-loading-ellipsis>`

File: src/components/row/readme.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+### mno-row settings
+
+* `name` -
+  The name of the link
+
+* `status` -
+  Additional information available at a glance.
+
+* `mno-row-href` or `mno-row-click` -
+  Exactly one of the mnoRowClick, mnoRowHref attributes are required to be specified.
```
Comment (alexnoox) [inline src/components/row/readme.md]: Should be `Only one of the mnoRowClick or mnoRowHref attributes are required to be specified.`

**Review by alexnoox — COMMENTED** [review]
- Please do not commit the dist directory (add it to .gitignore)
- Delete the file .yo-rc.json
- I'm not sure about the file: `src/mnoUiElements.js`, maybe we should directly use the module `mnoUiElements` everywhere.
- Question: Should we prefix variables and classes with `mno-`?

File: src/components/admin/readme.md (hunk 3 above)
Comment (clemoberti) [reply src/components/admin/readme.md:48]: This is a sub class of sidebar-list

File: README.md (hunk 2 above)
Comment (clemoberti) [reply README.md:15]: :+1: :grapes:

File: src/components/kpi/kpi.component.coffee (hunk 4 above)
Comment (clemoberti) [reply src/components/kpi/kpi.component.coffee]: :+1: +1

**Comment by clemoberti** [comment]
@alexnoox I fixed the couple of issues you mentioned.

**Review by alexnoox — APPROVED** [review]
LGTM

**Pull request merged**